<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;售楼&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="accessibleName">
      <string>楼盘信息</string>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="renyuan">
      <attribute name="title">
       <string>售楼人员登记管理</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>460</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>登记</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>楼盘编号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>楼号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>单元号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>270</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>房号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>310</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>户型</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>340</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>面积</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>230</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>楼层</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>110</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>150</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>190</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>230</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>270</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>310</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>350</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>用户</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>80</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="hetong">
      <attribute name="title">
       <string>售楼合同登记管理</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>490</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>登记</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>合同编号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>140</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>合同日期</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>170</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>购买人</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>200</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>身份证</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>230</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>售楼人</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>合同单价</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>290</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>付款方式</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>320</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>贷款银行</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>110</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_17">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>140</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_18">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>170</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_19">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>200</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_20">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>230</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_21">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>260</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_22">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>290</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_23">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>320</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="shoukuan">
      <attribute name="title">
       <string>收款信息登记管理</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>450</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>登记</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>90</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>收款编号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>120</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>交款日期</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_45">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>150</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>收款金额</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_46">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>180</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>交款人</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_47">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>210</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>收款人</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_40">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>90</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_41">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>120</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_42">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>150</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_43">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>180</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_44">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>210</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="renyuancha">
      <attribute name="title">
       <string>售楼人员信息查询</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>440</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>1031</width>
         <height>371</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>用户</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>楼盘编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>楼号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>新建列</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>单元号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>楼层</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>房号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>面积</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="hetongcha">
      <attribute name="title">
       <string>售楼合同信息查询</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>440</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>871</width>
         <height>351</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>合同编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>现存</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>合同日期</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>身份证号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>新建列</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>售楼人</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>合同单价</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>贷款银行</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="shoukuancha">
      <attribute name="title">
       <string>收款信息查询</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>450</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>961</width>
         <height>361</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>收款编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>现存</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>交款日期</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>收款金额</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>收款人</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="baobiao">
      <attribute name="title">
       <string>显示打印报表</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>420</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>报表</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>941</width>
         <height>351</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>用户</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>现存</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>楼盘编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>收款编号</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
